pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/etterlatte/maler/vedlegg/omstillingsstoenad/Beregningsperiodetabell.kt

```diff
@@ -67,6 +68,13 @@ data class Beregningsperiodetabell(
                                     Language.English to " - sanction",
                                 )
                             }
+                            showIf(periode.institusjon) {
```

Comment:
Mulig man bør høre med Mari her og det skal være mulig med dobbelt opp. For det kan vell i teorien være både sanksjon og institusjonsopphold?

Reply:
Siden sanksjon alltid går til 0 kroner, så kan vi vel bare sjekke at det ikke er sanksjon tenker jeg. Sjekker opp.

Reply:
Mari ville ha begge deler :)